src/main/components/urlBar.jsx

```diff
@@ -0,0 +1,53 @@
+import React, { useState } from 'react';
+
+const UrlBar = (props) => {
+  const method = (props.AorB === 'B' ? 'POST' : 'GET')
+  const [selected, setSelected] = useState(method);
+
+  function sendFetch () {
+    const sendingObj = { method: selected, mode: 'cors' };
+    if (props.SourceOrDest === 'dest') sendingObj.body = JSON.stringify(props.requestBody);
+
+    // console.log('hello;', document.querySelector('#urlInput').value);
+
+    fetch(document.querySelector('#urlInput').value, sendingObj)
+      //.then(r => r.json())
+      .then(r1 => console.log(r1));
+  }
+
+  console.log(props)
+
+  return (
+    <div>
+
+      <div > {/* onSubmit={sendFetch} */}
```

Comment:
Containing element should be a `form` here, which is more semantic _and_ allows us to use a `submit` event (which is more flexible than just a button with a `click` event)